components/connector/config/servers.go

```diff
@@ -1,9 +1,11 @@
 package config
 
 import (
+	"github.com/kyma-incubator/compass/components/director/pkg/log"
```

Comment:
The functions that you use from this package are introduced with your PR. Update the go.mod file.